src/InfRcTransport.cc

```diff
@@ -757,9 +757,19 @@ InfRcTransport::clientTrySetupQueuePair(IpAddress& address)
             inet_ntoa(sin->sin_addr), clientPort, nonce);
 
     for (uint32_t i = 0; i < QP_EXCHANGE_MAX_TIMEOUTS; i++) {
+        ibv_gid gid;
+        memset(&gid, 0, sizeof(gid));
+        // TODO: make gid index configurable!  This will not work in the general case.
+        const int gidIndex = 3;
```

Comment:
I know you worked hard on figuring out the correct gidIndex. Can you make this a static const instead of saying 3 everywhere?